Source/Moya.swift

```diff
@@ -285,3 +287,18 @@ private struct CancellableWrapper: Cancellable {
 
 /// Make the Alamofire Request type conform to our type, to prevent leaking Alamofire to plugins.
 extension Request: RequestType { }
+
+/// Make `NSURLRequest` conform to `RequestType` so that it can be sent with `PluginType` method `willSendRequest(_:target:)`.
+extension NSURLRequest: RequestType {
+    public var request: NSURLRequest? {
+      return self
+    }
+
+    public func authenticate(usingCredential credential: NSURLCredential) -> Self {
```

Comment:
These two methods make me feel a bit weird, since they don't do anything but should. I realize it's a bit of a chicken-and-the-egg problem :hatching_chick: Is this what @wangshengjia was talking about with calling `resume` manually?